### ISSUE TITLE
Monitor still runs against net5.0 which is unsupported and failing builds

### ISSUE THREAD

**Issue by heaths** [Azure/azure-sdk-for-net#32568]
The `net - monitor - ci` pipeline is failing because Monitor runs against net5.0, which is unsupported and causing failures:

> C:\hostedtoolcache\windows\dotnet\sdk\6.0.403\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): Error NETSDK1138: The target framework 'net5.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy.

net6.0 is the current LTS. If there's no specific reason to run against net5.0, I recommend sticking with the defaults passed from _eng/Directory.Build.Common.props_.

**Comment by TimothyMothra** [comment]
Specifically, `Azure.Monitor.Ingestion.csproj` is failing ci builds.
Can you ping someone from that team to look into this?

https://github.com/Azure/azure-sdk-for-net/blob/0f0b0bec375fd17754420124adad4d72b0243ad5/sdk/monitor/Azure.Monitor.Ingestion/src/Azure.Monitor.Ingestion.csproj#L7


Example failure: 
```
error NETSDK1138: The target framework 'net5.0' is out of support and will not receive security updates in the future. 
Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. 
[D:\a\_work\1\s\sdk\monitor\Azure.Monitor.Ingestion\src\Azure.Monitor.Ingestion.csproj]
```
https://dev.azure.com/azure-sdk/public/_build/results?buildId=2003985&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=45862c6e-83c5-515f-73e2-e7009eff9f9b&l=16

**Comment by jsquire** [comment]
I fixed this as part of #32607.
